ui.R

```diff
@@ -143,7 +195,8 @@ function(request) {
               )
             ),
             tabPanel(
-              "Quality",
+              value="quality",
```

Comment:
We need the equivalent lines to these adding for cost and forecast for those links to be able work as well. At the moment the links are pointing to undefined elements so aren't taking you to the right places.

Reply:
Needs to be `value="forecast",` at the start of the forecasting tabPanel() and `value="cost",` at the start of the Cost metric tabPanel().

Reply:
Updated